test/plugins/individual-collateral/origin/constants.ts

```diff
@@ -26,6 +26,7 @@ export const BASE_ORACLE_ERROR = fp('0.0015') // only using ETH/USD feed at the
 export const PRICE_TIMEOUT = bn('604800') // 1 week
 export const ORACLE_TIMEOUT = bn(86400) // 24 hours in seconds
 export const ORACLE_ERROR = fp('0.005')
+export const OETH_ORACLE_ERROR = fp('0.005')
```

Comment:
i think the error of this oracle is 1% not 0.5%

Reply:
Nice catch thanks, I fixed it in this commit: [06b939c](https://github.com/reserve-protocol/protocol/pull/1239/commits/06b939c408af0f5129c4404f590e5a58ec05250f)